src/Microsoft.NetCore.Analyzers/UnitTests/Security/ReviewCodeForDllInjectionVulnerabilitiesTests.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.NetCore.Analyzers.Security.UnitTests
+{
+    using System;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using Microsoft.CodeAnalysis.Testing;
+    using Microsoft.NetCore.Analyzers.Security;
+    using Test.Utilities;
+    using Xunit;
+
+    public class ReviewCodeForDllInjectionVulnerabilitiesTests : TaintedDataAnalyzerTestBase
```

Comment:
>ReviewCodeForDllInjectionVulnerabilitiesTests [](start = 17, length = 45)

Please add Traits attribute with dataflow and security traits. I normally only run unit tests with dataflow trait whenever modifying core flow analysis infrastructure/analyses.

Reply:
The base class, `TaintedDataAnalyzerTestBase`, has the `[Trait]` on it, and it Works On My Machine(TM).  Will add one for Security.

Reply:
Or...I'll figure out what I want to do for Security later.